### PR TITLE
HAL-1580: Support Global EJB Server Side Interceptors Configuration in Console

### PR DESCRIPTION
**Pull request by hpehl**
https://issues.jboss.org/browse/HAL-1580

### CODE REVIEW

File: console/HAL-1580_Global_EJB_Server_Side_Interceptors_Configuration.adoc [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+* {issue-base-url}/EAP7-474[EAP7-474]
+* {issue-base-url}/WFLY-6143[WFLY-6143]
+
+=== Dev Contacts
+
+* mailto:hpehl@redhat.com[Harald Pehl]
+
+=== QE Contacts
+
+* mailto:jkasik@redhat.com[Jan Kasik]
```
Comment (mnovak1) [inline console/HAL-1580_Global_EJB_Server_Side_Interceptors_Configuration.adoc]: QE contact will be Martin Simka.

File: console/HAL-1580_Global_EJB_Server_Side_Interceptors_Configuration.adoc [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+* WildFly
+* HAL
+
+== Requirements
+
+* Interceptors should be configured in the EJB3 subsystem
+
+== Test Plan
+
+TBD
```
Comment (mnovak1) [inline console/HAL-1580_Global_EJB_Server_Side_Interceptors_Configuration.adoc]: There is missing hight level overview which tests will be implemented.

File: console/HAL-1580_Global_EJB_Server_Side_Interceptors_Configuration.adoc [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+:email:             hpehl@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse
+
+== Overview
+
+The goal of this issue is to provide support in HAL to configure system-wide server side EJB interceptors through the ejb3 subsystem.
```
Comment (zhantemirov) [inline console/HAL-1580_Global_EJB_Server_Side_Interceptors_Configuration.adoc]: May I have a suggestion? I wonder whether it is possible to implement the following feature:

1. User needs to provide only path to an interceptor .jar - module is created automatically (so explicit EAP configuring and manual copying won't be needed).

Comment (hpehl) [reply console/HAL-1580_Global_EJB_Server_Side_Interceptors_Configuration.adoc]: That's a good suggestion. HAL can only use DMR operations to talk to the management layer. AFAIK there's no way to create a module using DMR.

File: console/HAL-1580_Global_EJB_Server_Side_Interceptors_Configuration.adoc [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+* mailto:hpehl@redhat.com[Harald Pehl]
+
+=== QE Contacts
+
+* mailto:jkasik@redhat.com[Jan Kasik]
+
+=== Testing By
+    
+[ ] Engineering
```
Comment (mnovak1) [inline console/HAL-1580_Global_EJB_Server_Side_Interceptors_Configuration.adoc]: This RFE will be tested by Engineering.

File: console/HAL-1580_Global_EJB_Server_Side_Interceptors_Configuration.adoc (hunk 1 above)
Comment (hpehl) [reply console/HAL-1580_Global_EJB_Server_Side_Interceptors_Configuration.adoc]: Done

File: console/HAL-1580_Global_EJB_Server_Side_Interceptors_Configuration.adoc (hunk 4 above)
Comment (hpehl) [reply console/HAL-1580_Global_EJB_Server_Side_Interceptors_Configuration.adoc]: Done

**Comment by hpehl** [comment]
Superseded by https://github.com/wildfly/wildfly-proposals/pull/433

**Pull request closed without merging**